spec/features/provider_login_spec.rb

```diff
@@ -9,7 +9,7 @@
     let(:email_address) { Faker::Internet.email }
 
     before do
-      sign_in create(:provider, email: email_address)
+      sign_in create(:provider, email: email_address, first_office_code: "LAA")
```

Comment:
This is very minor but can we change the 'LAA' code to something like 1Q630KL

Reply:
Agreed, have updated.